internal/handlers/common/params.go

```diff
@@ -391,6 +374,32 @@ func multiplyNumbers(v1, v2 any) (any, error) {
 	}
 }
 
+// multiplyLongSafely returns the multiplication of two int64 values.
+// It handles int64 overflows, and returns errLongExceeded error on one.
+//
+// Please prefer to use multiplyNumbers instead of this function.
```

Comment:
But when should we use this function if `multiplyNumbers` is preferred instead?

Reply:
I guess that only in `multiplyNumbers` as it's just a helper function for it

Reply:
Hmm, maybe we can rephrase the comment this way? Something like `Please always use multiplyNumbers as it calls multiplyLongSafely when needed.`

Reply:
fixed